README.md

```diff
@@ -72,6 +74,14 @@ Before you do that you will need to connect to the RDS console query editor. Use
 
 By following all the steps above you'll have a table in the database to store profile entries into talent board.
 
+**7.** To deploy API docs to your AWS account, follow the steps given below:
+```shell
+npm run apidoc # This will build the API documentation - creates the static HTML, CSS etc. 
+cd ApiDocs
+serverless --verbose # Deploys api documentation to your aws account
+cd ..
+```
+
```

Comment:
Add the following line after the step:

The above steps will deploy the statically generated API docs by running the `npm run apidoc` and the same gets deployed to an S3 bucket using the [Serverless Components](https://github.com/serverless-components/website)

Reply:
Addressed this remark